arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -605,6 +605,75 @@
         }
       }
     },
+    "UpgradeDetails": {
+      "description": "Details for upgrading vault.",
+      "type": "object",
+      "properties": {
+        "operationId": {
+          "description": "ID of the vault upgrade operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "startTimeUtc": {

```

Comment:
Confirmed with the service team that formatted date-time string is sent by the service. Taking this change.